src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -259,7 +259,7 @@ const styles = StyleSheet.create({
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `wrapperSearch` has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this change does not affect the visibility and contrast of the component against its background.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->